repair/repair.cc

```diff
@@ -1612,3 +1612,96 @@ future<> bootstrap_with_repair(seastar::sharded<database>& db, locator::token_me
         rlogger.info("bootstrap_with_repair: finished with keyspaces={}", keyspaces);
     });
 }
+
+future<> do_decommission_removenode_with_repair(seastar::sharded<database>& db, locator::token_metadata tm, gms::inet_address leaving_node) {
+    using inet_address = gms::inet_address;
+    return seastar::async([&db, tm = std::move(tm), leaving_node = std::move(leaving_node)] () mutable {
```

Comment:
A nitpick: inet_address is a uint32_t. You don't need to std::move() it.

Reply:
It is going to change, hello ipv6.

Reply:
Opps. Missed that. You are right.

Reply:
IPv6 doesn't benefit from move either :)

Reply:
(but we should make node identifiers lw_shared_ptr<node>, and that does benefit from move)